h2o-admissibleml/src/main/java/hex/Infogram/EstimateCMI.java

```diff
@@ -0,0 +1,44 @@
+package hex.Infogram;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+
+public class EstimateCMI extends MRTask<EstimateCMI> {
+  public int _nonZeroRows;
+  public double _accumulatedCMI;
+  public double _meanCMI;
+  public final double _scale = 1.0/Math.log(2);
+  public final int _responseColumn;
+
+  public EstimateCMI(Frame fr) {
+    _meanCMI = 0.0;
+    _responseColumn = fr.numCols()-1;
+  }
+
+  @Override
+  public void map(Chunk[] ck) {
+    _nonZeroRows = 0;
+    _accumulatedCMI = 0.0;
+    int numRow = ck[0].len();
+    for (int rowIndex = 0; rowIndex < numRow; rowIndex++) {
+      int prediction = (int) ck[_responseColumn].atd(rowIndex);
+      double predictionProb = ck[prediction+1].atd(rowIndex);
+      if (predictionProb > 0) {
```

Comment:
this seems suspicious - when does that ever happen? if the predicted class has 0 probability then that means all classes have 0 probability - which means we cannot really make a prediction

also - there can be NAs in response, this should be handled

Reply:
I need to do a log on predictionProb and log(0) is bad.  Added handling of NA results.  Thanks.